src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,108 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    public static final int DEFAULT_CAPACITY = 10;
+    public static final double GROW_FACTOR = 1.5;
+    private Object[] elementData;
+    private int size;
+
+    public ArrayList() {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line before the constructor to adhere to the coding standards.